common/src/main/java/dev/endoy/helpers/common/injector/Injector.java

```diff
@@ -301,16 +301,18 @@ private void validateConstructor( Constructor<?> constructor )
     {
         if ( !Arrays.stream( constructor.getParameters() ).allMatch( parameter -> this.isInjectable( parameter.getType() ) || parameter.isAnnotationPresent( Value.class ) ) )
```

Comment:
while we're at it, maybe it's even better to move the `nonInjectableParameters` to above this `if` statement so that we can re-use the list (just check `if ( !nonInjectableParameters.isEmpty() ) {`)

Reply:
solved